testsuite/tests/statmemprof/intern.byte.reference

```diff
@@ -1,15 +0,0 @@
-check_nosample
-check_counts_full_major
-check_counts_full_major
-check_no_nested
-check_distrib 2 3000 3 0.000010
-check_distrib 2 3000 1 0.000100
-check_distrib 2 2000 1 0.010000
-check_distrib 2 2000 1 0.900000
-check_distrib 300000 300000 20 0.100000
-check_callstack
-Raised by primitive operation at unknown location
-Called from file "intern.ml", line 32, characters 14-35
-Called from file "intern.ml", line 185, characters 2-25
-Called from file "intern.ml", line 191, characters 9-27
-OK !
```

Comment:
I would expect this file to be deleted, but it is unclear from the github diff that it is.

Reply:
It is indeed deleted.